src/client/src/components/Footer.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Footer = () => {
+    return (
+        <div class="w-100 p-3 bg-dark text-white text-center fixed-bottom">NGO-SWERA</div>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer

You can use `footer` element here.